### DISCUSSION TITLE
Is it possible to get this running on github.dev?

### DISCUSSION THREAD

**Discussion by Hellscaped** [community/community#32156] [Copilot Conversations] [answered]
You see, I use a school chromebook. I can't install vscode. I'm looking for a way to run copilot on github.dev.

**Comment by DaisyDogs07** [comment]
GitHub Copilot for the web seems to be available in github.dev codespaces, But not regular github.dev editors tho.
Hope this helps :)

**Comment by davecheney** [comment, marked as answer]
The short version is, no. 

The longer version is, it is something we’d like to do, but while vscode is just a giant blob of typescript, the environment it’s hosted in, the browser, vs electron (yes, I know electron is basically a browser, sorry, I don’t make the rules ;) means Copilot has grown to expect several, non trivial, things the electron/node.js environment provides. Finding alternatives to those, for example end to end http2 support—not to mention to evergreen problem for proxies—turns out to be a non trivial problem and so GitHub.dev support is not top fo the priority list at the moment.
